### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by ZPears** [Alchyr/BasicMod#7]
First time modder here - I'm on linux and when getting my first attempted mod running, I realized there wasn't a setting in the pom.xml for linux, and it defaulted to the settings for steam.mac. I was able to get it working by changing the steam.mac settings to the linux steam paths, but presumably this is not the way I'd actually want to release a mod.

Are there any recommendations from repo owner on how to add linux support? I'm happy to implement with a little guidance.

**Comment by Alchyr** [comment]
mind giving me an example of what the paths look like for steam on linux? You don't have to worry about releasing; the pom.xml and general setup are only relevant for the compilation of the mod into a `.jar`. Once you have a functional mod file, it'll work fine anywhere.

**Comment by ZPears** [comment]
Ah good to know - I haven't finished my mod yet so didn't know how much (if at all) this would matter for the release process.

The path for me on ubuntu was ~/.steam/debian-installation/steamapps/common/SlayTheSpire/desktop-1.0.jar (this should be the default - I didn't make any changes during my steam installation).

In my clone of the repo, I changed properties > steam.mac to ~/.steam/debian-installation/steamapps, and for profiles, I changed the mac one to be profiles > profile > properties > steam.path > sts.path to /common/SlayTheSpire/desktop-1.0.jar, and then it all worked fine.

Is there a steam.linux or similar?
